### DISCUSSION TITLE
what is the estimated Ebesucher Earning per IP?

### DISCUSSION THREAD

**Discussion by SkullFace141** [engageub/InternetIncome#22] [General]
what is the estimated Ebesucher Earning per IP?
IP is Residential from USA

**Comment by SkullFace141** [comment]
@engageub

**Comment by engageub** [comment]
Its about $0.10 per day per ip or more based on your IP score. Please make sure your IPs are on different subnet when multiple proxies are used. Datacenter may not work all the time and you would see captcha to be solved if IP score is bad. 
Please note that all the node earnings are based on your location and type of proxy. It varies. You will have to try it yourself. 
Thank you

**Comment by SkullFace141** [comment]
how to avoid the browser from getting stuck at a dead website?

**Comment by SkullFace141** [comment]
@engageub

**Comment by engageub** [comment]
The script autorestarts every hour. Please take a screenshot and attach here what you mean by dead website

Thank you

**Comment by SkullFace141** [comment]
sometimes a website is dead showing host not found error and it become stuck unless restarted or entering the surf url manually

**Comment by engageub** [comment]
The extension should take of those use cases. It is not related to this script.  If you hover the mouse over the page, it usually shows the waiting time on a website. It can go from 10 seconds to 5 minutes depending on the website. 

Thank you

**Reply by engageub** [reply]
Please send email to ebesucher support team if you see any issue. Please take a screenshot and send it to them with the issue. The script is only responsible to create multiple containers and redirect them through proxies. Any issue with the application may be result of the application itself. 

Thank you
